rotkehlchen/db/upgrades/v31_v32.py

```diff
@@ -85,6 +85,12 @@ def _add_new_tables(cursor: 'Cursor') -> None:
     FOREIGN KEY(parent_identifier) references history_events(identifier) ON UPDATE CASCADE ON DELETE CASCADE,
     PRIMARY KEY (parent_identifier, value)
 );""")  # noqa: E501
+    cursor.execute("""
+    CREATE TABLE IF NOT EXISTS ens_mappings (
+    address TEXT NOT NULL PRIMARY KEY,
+    ens_name TEXT UNIQUE
+);
+""")  # noqa: E501
```

Comment:
the `noqa: e501` is needed when you have a long line that you don't want to split. In this case I believe it can be removed

Reply:
Oh, sure, I don't now how it appeared there :)